code/divide_conquer/src/Search_in_a_Rotated_Array/Search_in_a_Rotated_Array.cpp

```diff
@@ -0,0 +1,118 @@
+/*
+Problem statement :
+Given a sorted and rotated array A of N distinct elements which is rotated 
+at some point, and given an element K. The task is to find the index of 
+the given element K in the array A.
+*/
+
+#include <bits/stdc++.h>
+using namespace std;
+#define ll long long int
+
+ll findPivot(ll arr[], ll low, ll high){
+    ll mid;
+
+    while(low < high){
+        // find mid element 
+        mid = low + (high-low)/2;
+
+        if(arr[mid] < arr[0]){
+            high = mid;
+        }
+        else{
+            low = mid + 1;
+        }
+    }
+
+    if(arr[low] < arr[0]){
+        return low;
+    }
+    else{

```

Comment:
Reformat this file using clang format with 4 spaces.

Reply:
Okay sir.